code/game/objects/effects/spawners/gibspawner.dm

```diff
@@ -12,7 +12,7 @@
 		gibamounts = list(1,1,1,1,1,1,1)
 
 		New()
-			gibdirections = list(list(NORTH, NORTHEAST, NORTHWEST),list(SOUTH, SOUTHEAST, SOUTHWEST),list(WEST, NORTHWEST, SOUTHWEST),list(EAST, NORTHEAST, SOUTHEAST), alldirs, alldirs, list())
+			gibdirections = list(list(NORTH, NORTHEAST, NORTHWEST),list(SOUTH, SOUTHEAST, SOUTHWEST),list(WEST, NORTHWEST, SOUTHWEST),list(EAST, NORTHEAST, SOUTHEAST), GLOB.alldirs, GLOB.alldirs, list())

```

Comment:
Inconsistent spessing after commas, but does it even matter since it indicates some separation b/w lists better than if it was consistent? :thinking: